.github/workflows/ci.yml

```diff
@@ -10,10 +10,22 @@ jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - run: npm install
-      - uses: lannonbr/vsce-action@master
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Comment:
```suggestion
          node-version: 20
          cache: "npm"
```

Reply:
Too slow! Ok but it failed immediately anyway, new PR incoming..

Reply:
I suspect that this repo will only have builds like, once every 18 months. But doesn't hurt..